src/board_controller/openbci/novaxr.cpp

```diff
@@ -94,6 +94,28 @@ int NovaXR::config_board (char *config)
         safe_logger (spdlog::level::err, "Failed to config a board");
         return BOARD_WRITE_ERROR;
     }
+    if (!is_streaming)
+    {
+        char response = 0;
+        res = socket->recv (&response, 1);
```

Comment:
Is there a timeout for this?

Reply:
yes, the same as for data packages(5 seconds)